extensions/database/src/com/google/refine/extension/database/mysql/MySQLDatabaseService.java

```diff
@@ -52,7 +52,7 @@ public class MySQLDatabaseService extends DatabaseService {
 
     private static final Logger logger = LoggerFactory.getLogger("MySQLDatabaseService");
     public static final String DB_NAME = "mysql";
-    public static final String DB_DRIVER = "com.mysql.jdbc.Driver";
+    public static final String DB_DRIVER = "com.mysql.cj.jdbc.Driver";
```

Comment:
```suggestion
    public static final String DB_DRIVER = com.mysql.cj.jdbc.Driver.class.getCanonicalName();
```

I dislike hiding class names in strings unless there's no choice. Is there any reason for us to not just use the class reference so that our tooling can help us track dependencies?

(I understand that this problem was not introduced by your PR)